Tools/TestWebKitAPI/CMakeLists.txt

```diff
@@ -345,10 +348,7 @@ if (ENABLE_WEBKIT)
     target_link_libraries(TestWebKitAPIBase PRIVATE WebKit::WebKit WebKit::gtest)
 
     if (COMPILER_IS_GCC_OR_CLANG)
-        WEBKIT_ADD_TARGET_CXX_FLAGS(TestWebKitAPIBase -Wno-dangling-else
-                                                      -Wno-sign-compare
-                                                      -Wno-undef
-                                                      -Wno-unused-parameter)
+        WEBKIT_ADD_TARGET_CXX_FLAGS(TestWebKitAPIBase ${TestWebKitAPI_DISABLED_WARNINGS} -Wno-deprecated-declarations)
```

Comment:
Looks like this didn't actually work; putting `-Wno-deprecated-declarations` inside `TestWebKitAPI_DISABLED_WARNINGS` is one possible solution.